File: 1.py
def tambah(a, b):
    return a + b

def kurang(a, b):
    return a - b

def kali(a, b):
    return a * b

def bagi(a, b):
    if b != 0:
        return a / b
    else:
        return "Tidak bisa dibagi dengan nol."

def kalkulator():
    print("=== Program Kalkulator Sederhana ===")
    while True:
        print("\nPilih operasi:")
        print("1. Penjumlahan")
        print("2. Pengurangan")
        print("3. Perkalian")
        print("4. Pembagian")
        print("5. Keluar")
        
        pilihan = input("Masukkan pilihan (1/2/3/4/5): ")

        if pilihan == '5':
            print("Terima kasih telah menggunakan program.")
            break

        angka1 = float(input("Masukkan angka pertama: "))
        angka2 = float(input("Masukkan angka kedua: "))

        if pilihan == '1':
            hasil = tambah(angka1, angka2)
            operasi = "Penjumlahan"
        elif pilihan == '2':
            hasil = kurang(angka1, angka2)
            operasi = "Pengurangan"
        elif pilihan == '3':
            hasil = kali(angka1, angka2)
            operasi = "Perkalian"
        elif pilihan == '4':
            hasil = bagi(angka1, angka2)
            operasi = "Pembagian"
        else:
            print("Pilihan tidak valid.")
            continue

        print(f"Hasil {operasi} dari {angka1} dan {angka2} adalah: {hasil}")

kalkulator()

File: 1.php
<!DOCTYPE html>
<html>

<head>
    <title>Kalkulator Sederhana</title>
</head>

<body>
    <h2>Kalkulator Sederhana</h2>
    <form method="post" action="">
        <label>Angka Pertama:</label><br>
        <input type="number" name="angka1" required><br><br>

        <label>Angka Kedua:</label><br>
        <input type="number" name="angka2" required><br><br>

        <label>Operasi:</label><br>
        <select name="operasi">
            <option value="tambah">Penjumlahan</option>
            <option value="kurang">Pengurangan</option>
            <option value="kali">Perkalian</option>
            <option value="bagi">Pembagian</option>
        </select><br><br>

        <input type="submit" name="hitung" value="Hitung">
    </form>

    <?php
    function tambah($a, $b)
    {
        return $a + $b;
    }

    function kurang($a, $b)
    {
        return $a - $b;
    }

    function kali($a, $b)
    {
        return $a * $b;
    }

    function bagi($a, $b)
    {
        if ($b == 0) {
            return "Tidak bisa dibagi nol.";
        } else {
            return $a / $b;
        }
    }

    if (isset($_POST['hitung'])) {
        $a = $_POST['angka1'];
        $b = $_POST['angka2'];
        $op = $_POST['operasi'];

        switch ($op) {
            case 'tambah':
                $hasil = tambah($a, $b);
                break;
            case 'kurang':
                $hasil = kurang($a, $b);
                break;
            case 'kali':
                $hasil = kali($a, $b);
                break;
            case 'bagi':
                $hasil = bagi($a, $b);
                break;
        }

        echo "<h3>Hasil: $hasil</h3>";
    }
    ?>
</body>

</html>